docs/usage/self-hosted-configuration.md

```diff
@@ -313,19 +313,19 @@ For example, if you set `dockerChildPrefix=myprefix_` then the final container c
 
 ## dockerImagePrefix
 
-By default Renovate pulls the sidecar Docker containers from `docker.io/renovate`.
+By default Renovate pulls the sidecar Docker containers from `docker.io/containerbase`.
```

Comment:
Is shifting to ghcr in future also a possibility?

Reply:
yes, the containerbase images are also published to ghcr.io

Reply:
Probably best to keep it with Docker Hub for now though as we know they work perfectly well until now (rate limits aside)

Reply:
i can add a note that you can also use the ghcr for official sidecar image